v2/tools/generator/internal/astbuilder/maps.go

```diff
@@ -12,10 +12,13 @@ import (
 )
 
 // MakeMap returns the call expression for making a map
-//
-// 	make(map[<key>]<value>)
-//
 func MakeMap(key dst.Expr, value dst.Expr) *dst.CallExpr {
+	/*
```

Comment:
The gofmt documentation suggests that it supports code blocks, as long as they're formatted "correctly": See https://tip.golang.org/doc/comment - section about "code blocks" (can't link to it for some reason).

For example check out: https://go.dev/play/p/zPOvetWAPA1 - if you click format on that, it adds a space between the top level method text and the code block, but leaves it alone other than that. So I think all of these places where we've moved the code block into the method can be done in that style and kept in the comment (which IMO looks better?)

I think you can get `go fmt` to do that automatically by just running the latest version of `go fmt` on the original set of files, at least for these `astbuilder` files where the issue was code block formatting.